### PR TITLE
SPLICE-1714: Ignore 'should not give a splitkey which equals to startkey' exception(2.5)

### PR DESCRIPTION
**Pull request by jyuanca**

### CODE REVIEW

File: hbase_storage/src/main/java/com/splicemachine/access/hbase/H10PartitionAdmin.java [hunk 1]
```diff
@@ -124,6 +141,18 @@ private void retriableSplit(TableName tableInfo, byte[] splitPoint) throws IOExc
                 } catch (InterruptedException ie) {
                     throw new InterruptedIOException();
                 }
+            } catch (IOException e) {
+                String errMsg = e.getMessage();
+                if (errMsg.compareTo("should not give a splitkey which equals to startkey!") == 0) {
```
Comment (snowspeeder) [inline hbase_storage/src/main/java/com/splicemachine/access/hbase/H10PartitionAdmin.java]: since you changed the message, this might need change

Comment (jyuanca) [inline hbase_storage/src/main/java/com/splicemachine/access/hbase/H10PartitionAdmin.java]: No, this is from HBase.

Comment (jyuanca) [inline hbase_storage/src/main/java/com/splicemachine/access/hbase/H10PartitionAdmin.java]: This is an HBase error message

**Comment by snowspeeder** [comment]
Oh, ok, sorry got confused.

**Pull request merged**